### PR TITLE
Enhancement: Add PHP 7.4 stable build which is not allowed to fail

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by weaverryan** [comment]
The problem is the MAKER_TEST_VERSION=dev should be allowed to fail.

So, best would be to make a new matrix item for just PHP 7.4, but then we take more CI resources from the rest of the symfony org.

File: .travis.yml [hunk 1]
```diff
@@ -21,11 +21,13 @@ matrix:
...
         - php: 7.3
           env: MAKER_TEST_VERSION=stable
+        - php: 7.4
+          env: MAKER_TEST_VERSION=stable          
         - php: 7.4
           env: MAKER_TEST_VERSION=dev
     allow_failures:
         - php: 7.4
-          env: MAKER_TEST_VERSION=dev
+          env: MAKER_TEST_VERSION=dev
```
Comment (OskarStark) [inline .travis.yml:30]:
```suggestion
          env: MAKER_TEST_VERSION=dev
```

**Comment by OskarStark** [comment]
I updated the code and the PR title accordingly 👍

**Comment by OskarStark** [comment]
> So, best would be to make a new matrix item for just PHP 7.4, but then we take more CI resources from the rest of the symfony org.

I will tackle this problem in #693

**Review by jrushlow — APPROVED** [review]

**Comment by OskarStark** [comment]
Superseded by #701

**Pull request closed without merging**